xbmc/network/upnp/UPnPPlayer.cpp

```diff
@@ -523,6 +517,18 @@ void CUPnPPlayer::DoAudioWork()
                                                  static_cast<void*>(new CFileItem(*item)));
     }
 
+    // Player may be paused or resumed from the target player, state needs to be synchronized to data cache core.
+    if (IsPaused() && CDataCacheCore::GetInstance().GetSpeed() > 0.0f)
```

Comment:
Not sure you want to keep `CDataCacheCore::GetInstance()` in a variable? You're calling it multiple times in this method.

Reply:
Yeah good suggestion, will do